README.md

```diff
@@ -18,13 +18,18 @@ Either install via `npm i -g xliff-simple-merge` or run directly with `npx xliff
 
 ```text
 Options:
-  -i, --input-file <inputFile>              input file/merge origin
+  -i, --input-file <inputFiles...>          input file(s)/merge origin(s)
   -d, --destination-file <destinationFile>  merge destination
   -o, --output-file <outputFile>            output file, if not provided "merge destination" is overwritten
+  -e, -exclude-file <excludeFiles...>       exclude all unit IDs of the provided file(s) 
   --no-match-fuzzy                          prevent fuzzy matching of similar units with changed id
   --no-collapse-whitespace                  prevent collapsing of multiple whitespaces when comparing translations sources
   --no-reset-translation-state              prevent (re-)setting the translation state to new/initial for new/changed units
   --no-replace-apostrophe                   prevent replacing of apostrophes (') with "&apos;"
   --debug                                   enable debug output
   -h, --help                                display help for command
 ```
+
+### Notes
+* If different input files contains the same unit IDs, only the latter ones will be used.
+* The input file can have entry units and if it is the case they will have priority.
```

Comment:
1. typo in "entry units"?
2. why should we do this?

Reply:
1. I'll change for "target in the units"
2. The priority for several input filess is not important, since the user can change the order of the input files as he want. What is important is to document it accordingly. I find giving priority to the tatter ones is more instinctive as it seems they would override eachother in a row - but it can be changed.
The priority of the input target over output target is also discutable. I find it more instinctive this way as we want to "apply a new input fine to an outut file", but it can be changed. Most importantely the output is conform and the behaviour documented.

Reply:
I'm not sure why there should be priority to units with targets. Regularly all files merged should either all have targets or all have no targets. Do I miss some scenario here? In case not, I think we should not further complicate the logic and just take the last one (and remove the 2nd note)?

Reply:
I changed the logic to the other way around